terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperView.java

```diff
@@ -123,5 +124,11 @@ protected void prepareResponse(HttpServletRequest request,
         super.prepareResponse(request, response);
         response.setCharacterEncoding(StandardCharsets.UTF_8.name());
     }
-
+    
+    private String date2LocalDateString(final Date date) {
+        DateTimeFormatter fmt = DateTimeFormatter.ofPattern("uuuu/MM/dd");
+        return date==null ? "" : fmt.format(Instant.ofEpochMilli(date.getTime())
```

Comment:
- [ ] It appears that the source code is not formatted.
Please obtain the format file from https://github.com/terasolunaorg/terasoluna-gfw/blob/master/eclipse/formatter.xml and apply the formatting.

Reply:
It is fixed by specified formatter.xml